packages/app/src/cli/utilities/developer-platform-client.ts

```diff
@@ -295,3 +298,26 @@ export interface DeveloperPlatformClient {
   devSessionDelete: (input: Omit<DevSessionOptions, 'assetsUrl'>) => Promise<DevSessionDeleteMutation>
   getCreateDevStoreLink: (input: string) => Promise<string>
 }
+
+export function createUnauthorizedHandler(client: DeveloperPlatformClient): UnauthorizedHandler {
+  let unauthorizedRetriesUsed = 0
+  return {
+    type: 'token_refresh',
+    handler: async () => {
+      unauthorizedRetriesUsed++
+      if (unauthorizedRetriesUsed > 1) {
```

Comment:
I think this is now covered in `graphql.ts` -- we only try to retry once per request